app/src/org/commcare/utils/SyncDetailCalculations.java

```diff
@@ -29,22 +30,26 @@
     public static void updateSubText(final CommCareHomeActivity activity,
                                      SquareButtonViewHolder squareButtonViewHolder,
                                      HomeCardDisplayData cardDisplayData) {
+
         SqlStorage<FormRecord> formsStorage = CommCareApplication._().getUserStorage(FormRecord.class);
         int numUnsentForms = formsStorage.getIDsForValue(FormRecord.META_STATUS, FormRecord.STATUS_UNSENT).size();
+
+        Pair<Long, String> lastSyncTimeAndMessage = getLastSyncTimeAndMessage();
+
         if (numUnsentForms > 0) {
             Spannable syncIndicator = (activity.localize("home.sync.indicator",
-                    new String[]{String.valueOf(numUnsentForms), cardDisplayData.text}));
+                    new String[]{String.valueOf(numUnsentForms)}));
             squareButtonViewHolder.subTextView.setText(syncIndicator);
-            squareButtonViewHolder.subTextView.setTextColor(activity.getResources().getColor(cardDisplayData.subTextColor));
```

Comment:
In the case where the subtext was being set based upon unsent forms, we were never checking to see if the background color should be the warning color, even though the implementation of `isSyncStronglyNeeded()` implies that we want to set the warning background when there are a lot of unsent forms.